### ISSUE TITLE
API /reports/activity mismatch

### ISSUE THREAD

**Issue by trentontrees** [grokability/snipe-it#9518]
#### Please confirm you have done the following before posting your bug report:

- [no] I have enabled debug mode 
- [yes] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
Documentation on https://snipe-it.readme.io/reference#reportsactivity specifies that the params are to be sent in the body (item_type,item_id).

I'm seeing that they need to be supplied via query params.  The history page of an asset is also sending the query with item_type and item_id as query param.

**To Reproduce**
1. send an API request to {{snipe}}/api/v1/reports/activity with item_type=asset&item_id={{XXX}}
2. You should see the same payload as presented on {{snipe}}/hardware/{{XXX}}#history

**Expected behavior**
The history of the asset should match between the webpage and the API

**Screenshots**
n/a

**Server (please complete the following information):**
 - Snipe-IT Version v5.1.4
 - OS: CentOS
 - Web Server: Apache
 - PHP Version: 7.3.27 

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser: Firefox
 - Version: 88

**Comment by StarlessNights** [comment]
True, the API happily takes the parameters as query parameters, but I cannot reproduce your error. It works just as fine with JSON payload.

Maybe you forgot to add the `Content-Type` header?
